package.json

```diff
@@ -28,21 +32,21 @@
   "dependencies": {},
   "devDependencies": {
     "quill": "^2.0.0-dev.3",
```

Comment:
Now Quill 2.0.0 has released, Can you upgrade to the version of Quill 2.0.0? Thank you!

Reply:
Updated!!